src/ThreeEditor/examples/ex6.json

```
@@ -0,0 +1,652 @@
+						"visible": false
+					},
+					{
+						"colorHex": 0,
+						"geometryData": {
+							"geometryType": "HollowCylinderGeometry",
+							"parameters": {
+								"depth": 1,
+								"innerRadius": 0,
+								"radius": 4
+							},
+							"position": [0,0,-4.5],
+							"rotation": [0,0,0]
+						},
+						"name": "OuterCylinder",
+						"type": "CylinderFigure",
+						"uuid": "ed9aeea8-3f0e-46a9-8129-2c79b03cecb0",
+						"visible": false
+					},
+					{
+						"colorHex": 0,
+						"geometryData": {
+							"geometryType": "HollowCylinderGeometry",
+							"parameters": {
+								"depth": 1,
+								"innerRadius": 0,
+								"radius": 2
+							},
+							"position": [0,0,-4.5],
+							"rotation": [0,0,0]
+						},
+						"name": "InnerCylinder",
+						"type": "CylinderFigure",
+						"uuid": "920e87c2-6760-4f6b-a6d8-371021f1fa6d",
+						"visible": false
+					}],
+				"metadata": {
+					"generator": "FigureManager.toJSON",
+					"type": "Manager",
+					"version": "0.12"
+				},
+				"name": "Figure Manager",
+				"type": "FigureManager",
+				"uuid": "0F0F0F0F-0F0F-0F0F-0F0F-0F0F0F0F0F0F"
+			},
+			"hash": "ee6e2092bb9fa5cd8e6384f3d06c1f53e7fe55aa",
+			"materialManager": {
+				"materials": [
+					{
+						"color": 7274496,
+						"density": 11.35,
+						"icru": 82,
+						"name": "LEAD",
+						"sanitizedName": "Pb",
+						"uuid": "261ee324-eb32-4a60-8097-6a686feaf082"
+					},
+					{
+						"color": 8454143,
+						"density": 0.00120479,
+						"icru": 104,
+						"name": "AIR, DRY (NEAR SEA LEVEL)",
+						"opacity": 0.05,
+						"sanitizedName": "air_dry_near_sea_level",
+						"transparent": true,
+						"uuid": "46780184-dd64-47cb-bcca-ec7acb18c940"
+					},
+					{
+						"color": 33023,
+						"density": 1,
+						"icru": 276,
+						"name": "WATER, LIQUID",
+						"opacity": 0.05,
+						"sanitizedName": "water_liquid",
+						"uuid": "4458544c-b173-49d3-ac4e-62e735cd86eb"
+					}],
+				"metadata": {
+					"generator": "MaterialManager.toJSON",
+					"type": "Manager",
+					"version": "0.12"
+				},
+				"name": "Material Manager",
+				"selectedMaterials": {
+					"261ee324-eb32-4a60-8097-6a686feaf082": 1,
+					"4458544c-b173-49d3-ac4e-62e735cd86eb": 4,
+					"46780184-dd64-47cb-bcca-ec7acb18c940": 2
+				},
+				"type": "MaterialManager",
+				"uuid": "0F0F0F0F-0F0F-0F0F-0F0F-0F0F0FFFFFFF"
+			},
+			"metadata": {
+				"generator": "YaptideEditor.toJSON",
+				"type": "Editor",
+				"version": "0.12"
+			},
+			"physic": {
+				"availableStoppingPowerFiles": {},
+				"enableNuclearReactions": true,
+				"energyLoss": 0.03,
+				"energyModelStraggling": "Vavilov",
+				"multipleScattering": "Moliere",
+				"stoppingPowerTable": "ICRU91"
+			},
+			"project": {
+				"description": "Water phantom, lead collimator and beam of protons",
+				"history": {
+					"redos": [],
+					"undos": []
+				},
+				"simulator": "fluka",
+				"title": "Beam of photons with lead collimator",
```

```suggestion
				"title": "Beam of protons with lead collimator",
```

There is fundamental difference between photons and protons

---

Photons are not supported by SHIELD-HIT12A as beam source